python/fastsim/demos/timing_demo.py

```diff
@@ -56,7 +56,7 @@
 
 veh_csv_path = fsim.package_root() / "resources/vehdb/2012_Ford_Fusion.csv"
 t0 = time.perf_counter()
-veh_python = fsim.vehicle.Vehicle.from_file(str(veh_csv_path), to_rust=True)
+veh_python = fsim.vehicle.Vehicle.from_file(str(veh_csv_path), to_rust=True).to_rust()
```

Comment:
@calbaker @robinsteuteville I would think this line should be:
`veh_python = fsim.vehicle.Vehicle.from_file(str(veh_csv_path))`

Am I misunderstanding something?

Reply:
Ah, I see, I think I was misunderstanding this part of the demo - both lines instantiate `RustVehicle`s. I think we ought to update the benchmark results starting on line 51 to reflect the new code.

Reply:
@kylecarow, could you see what needs to be done to make this PR mergeable?

Reply:
@calbaker I reviewed all changes and seems that it's ready to merge as-is